tools/monorepo-utils/src/md-docs/lib/__tests__/generate-frontmatter.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * Internal dependencies
+ */
+import { generatePostFrontMatter } from '../generate-frontmatter';
+
+describe( 'generateFrontmatter', () => {
+	it( 'should not allow disallowed attributes', () => {
+		const frontMatter = generatePostFrontMatter( `---
+title: Hello World
+description: This is a description
+post_content: This is some content
+post_title: This is a title
+---
+` );
+
+		expect( frontMatter ).toEqual( {
+			post_title: 'This is a title',
+		} );
+	} );
+
+	it( 'should not do additional date parsing', () => {
```

Comment:
What kind of date parsing are you making sure doesn't happen here? Feels like there might be an instructive anecdote to this?

Reply:
Good idea. Its js yaml parsing that changes the date. I wanted to allow dates to remain raw for now until we have feedback on whether thats an issue for content authors.

Ill add a more descriptive comment.

Reply:
Resolved [here](https://github.com/woocommerce/woocommerce/pull/39214/commits/e06fff0903935df14b39d64eb070fe2c35d0ccba)